server/controllers/importers/graphml.js

```diff
@@ -2,7 +2,7 @@ var constants = require(__dirname + "/../constants");
 var parseString = require("xml2js").parseString;
 var semanticChecker = require(__dirname + "/../semantic-checker");
 
-/* 
+/*
```

Comment:
Per our emerging style conventions, can we just delete this block of code rather than commenting it out? We can always retrieve from `git` in case we want this back.